README.md

```diff
@@ -312,6 +312,50 @@ user=> (sci/eval-string "(with-out-str (foo))" {:bindings {'foo wrapped-foo}})
 "yello!\n"
 ```
 
+**Important note:** despite what was said [above](#usage), forms evaluated by sci that produce
+side effects via bindings (e.g. `println`) **can**, in some cases, "escape" the evaluation
+context of `sci/eval-string`. Specifically if those side-effecty functions are invoked outside
+`sci/eval-string` due to later realisation of a lazy sequence returned from it.  For example,
+the following code does not work:
+
+````clojure
```

Comment:
I just noticed that this line has 4 backticks instead of 3 - that's what's messing up the formatting.